packages/cdk8s-cli/templates/java-app/pom.xml

```diff
@@ -0,0 +1,40 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.mycompany.app</groupId>
+  <artifactId>my-app</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0-SNAPSHOT</version>
+  <name>my-app</name>
+  <url>http://maven.apache.org</url>
+  <dependencies>
+    <dependency>
```

Comment:
Is this needed here? Aren’t we installing explicitly in the post script?

Perhaps the alternative to installing from the script (in the non-local case) is to just plug in the right version number here and let maven work it’s magic. 

Same for `constructs`. We want the version to come from the CLI

Reply:
Looked into it a bit, I couldn't find a `mvn` command similar to `npm install --save` (nothing will _write_ a new dependency to the `pom.xml`). That's why I added the `mvn versions:use-dep-version` step.

We could do something like:

```
    <dependency>
      <groupId>org.cdk8s</groupId>
      <artifactId>cdk8s</artifactId>
      <version>x.y.z</version>
    </dependency>
```

If we want it to be really clear it's a placeholder.

Reply:
For `constructs` and `jsii`, I don't think that should be configurable from the CLI when running `cdk8s init`. I think `cdk8s` version makes sense here as it's a `cdk8s` project. But as for specifying versions of other dependencies.. I'm fine with letting folks change that post-init on their own.

Reply:
Can we just edit the xml file during the post script and plug in the version?

Reply:
Done!